src/stores/security-google-recaptcha.md

```diff
@@ -62,138 +56,111 @@ Google reCAPTCHA requires a pair of API keys to enable. You can get these keys f
-    [_Google reCAPTCHA keys_]({% link configuration/security/google-recaptcha.md %})
+      - Choose the **Theme** that is used to style the Google reCAPTCHA box. Options: Light Theme (default) / Dark Theme
 
-1. Select *reCaptcha type* as _Invisible reCaptcha v3_.
+      - Enter a two-character **[Language Code](https://developers.google.com/recaptcha/docs/language)** that specifies the language that is used for Google reCAPTCHA text and messaging.
 
-    ![Google reCAPTCHA v2 - type configuration]({% link images/images/config-security-recaptcha-type-v3.png %}){: .zoom}
+      - In the **reCAPTCHA Validation Failure Message** field, enter the message to appear if validation fails. Default: You cannot proceed with such operation, your reCAPTCHA reputation is too low.
 
-1. Expand (![Expansion selector]({% link images/images/btn-expand.png %})) the **Backend** section and set the options.
+   ![]({% link images/images/config-security-google-recaptcha-admin-v3-invisible.png %}){: .zoom}
+   [_reCAPTCHA v3 Invisible_]({% link configuration/security/google-recaptcha-admin.md %})
 
-   If needed, clear the **Use system value** checkbox to change the setting.
+1. Expand the **Admin Panel** section.
 
-   - **Enable** — Set to `Yes`.
+1. Clear the **Use system value** checkbox for each field that you want to configure.  Then, configure the following fields as needed.
 
-   - **Minimum score** — Input a value from `0.0` to `1.0`, where `1.0` is very likely a good interaction, and `0.0` is very likely a bot. For more, details see the official product documentation [Interpreting the score][3]
+   - Set **Enable for Login** to the reCAPTCHA type that you want to use for the Admin Sign In page.
 
-    ![Google reCAPTCHA - backend configuration]({% link images/images/config-2fa-google-recaptcha-backend-v3.png %}){: .zoom}
+   - Set **Enable for Forgot Password** to the reCAPTCHA type that you want to use for password reset requests.
 
-1. Expand (![Expansion selector]({% link images/images/btn-expand.png %})) the **Frontend** section and add Google reCAPTCHA to customer accounts:
+![]({% link images/images/config-security-google-recaptcha-admin.png %}){: .zoom}
+[_Admin Panel_]({% link configuration/security/google-recaptcha-admin.md %})
 
-    ![Google reCAPTCHA - frontend configuration]({% link images/images/config-2fa-google-recaptcha-v3-frontend.png %}){: .zoom}
-    [_Google reCAPTCHA Frontend_]({% link configuration/security/google-recaptcha.md %})
+## Step 3: Configure Google reCAPTCHA Storefront
 
-    If needed, clear the **Use system value** checkbox to change the setting.
+1. In the left panel under _Security_, choose **Google reCAPTCHA Storefront**.
 
-   - **Enable** — Set to `Yes`.
+1. Complete the section for each reCAPTCHA type that you want to use in the storefront. See the previous instructions for information about each option.
```

Comment:
maybe link previous instructions to the section header?

Reply:
I think it's OK as is. it's just the previous step.